ibis/common/patterns.py

```diff
@@ -1647,6 +1711,9 @@ class Topmost(Slotted, Pattern):
     """Traverse the value tree topmost first and match the first value that matches."""
 
     __slots__ = ("pattern", "filter")
+    pattern: Pattern
+    # Used Any as not sure how to type this
+    filter: Any
```

Comment:
`Any` is actually a pattern defined in this file, I assume these should be `AnyType`

Reply:
Done